src/scripts/main-nav.js

```diff
@@ -117,18 +117,27 @@
     }
   }
 
-  function getLoginElem (nickname, avatarImageURL) {
+  function getLoginElem (id, nickname, githubAvatar) {
+    if (nickname) {
+      // github user so show their nickname and avatar
+      var avatarImageURL = githubAvatar;
+      var label = nickname;
+    } else {
+      // email/password user so just show 'account' and default avatar
+      var avatarImageURL = 'common/styleguide/images/default-avatar.png';
+      var label = 'Account';
+    }
```

Comment:
You could even simplify this some:

```
    // email/password user so just show 'account' and default avatar
    var avatarImageURL = 'common/styleguide/images/default-avatar.png';
    var label = 'Account';

    if (nickname) {
      // github user so show their nickname and avatar
      avatarImageURL = githubAvatar;
      label = nickname;
    }
```
Is it possible for someone to have a `nickname` but no `githubAvatar`?